src/index.html

```diff
@@ -2,15 +2,43 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      class="searchForm"
+      action="/"
+      method="get"
+      data-qa="big"
+    >
+      <label for="searchInputBig"></label>
+      <input type="text"
+        id="searchInputBig"
+        class="searchInput"
+        name="searchInput"
+        placeholder="Try &quot;Los Angeles&quot;"
+        data-qa="keypress" required
+      >
```

Comment:
```suggestion
      <input
        type="text"
        id="searchInputBig"
        class="searchInput"
        name="searchInput"
        placeholder="Try &quot;Los Angeles&quot;"
        data-qa="keypress" 
        required
      >
```

Reply:
fix the previous comments
https://github.com/mate-academy/layout_search-bar-airbnb/pull/3121/files#r1145215078